README.md

```diff
@@ -30,6 +30,11 @@ class StartScanningViewController: UIViewController {
             print(error)
         }
     }
+
+    // In addition to `success`, `warning`, and `error` results, `SplitScreenScanner` allows a `pending` result to be passed to it's delegate `didScanBarcode` methods. Pending scans are useful for when a network task needs to be performed before determing if the scan was successful. They will not be displayed in the table. It is recommended when dealing with background tasks to block the scanner while the task is ongoing. Then when the task completes, unblock the scanner and manually add the appropriate `success`, `warning`, or `error` result to the table using the below `addScanResultToTable` method.
```

Comment:
- `determining`
- I feel like `below` is not needed in the sentence `Then when the task completes, unblock the scanner and manually add the appropriate success, warning, or error result to the table using the below addScanResultToTable method.`

Reply:
I think the way erik reworded this below actually works well, since it makes clear what the function in the example is useful for, so if you are good with keeping that last sentence as worded there, I think I'll leave it

Reply:
I'm down for using the copy that Erik reworded